tests/integration/test_unary_functions_2d.py

```diff
@@ -14,74 +14,78 @@
 #
 
 import numpy as np
+import pytest
 
 import cunumeric as num
 
+xn = np.array([[1, 2, 3], [4, 5, 6]])
 
-def test():
-    xn = np.array([[1, 2, 3], [4, 5, 6]])
+
+def test_sin():
     x = num.array(xn)
-    # print(np.sin(xn))
-    # print(num.sin(x))
     assert np.allclose(np.sin(xn), num.sin(x))
 
-    # print(np.cos(xn))
-    # print(num.cos(x))
+    for xx in [-10 - 1, -0.5, 0, 0.5, 1, 10]:
+        assert np.allclose(num.sin(xx), np.sin(xx))
```

Comment:
All of the scalar tests in this module seemed to be missing various corner cases, so I expanded the list of values that are looped over in each case